server/protocols/jmap/src/main/java/org/apache/james/jmap/model/MessageFactory.java

```diff
@@ -160,6 +160,7 @@ private Attachment fromMailboxAttachment(org.apache.james.mailbox.store.mail.mod
         return Attachment.builder()
                     .blobId(attachment.getAttachmentId().getId())
                     .type(attachment.getType())
+                    .name(attachment.getName().orNull())
```

Comment:
it's probably the build() method that is supposed to do this default value choice, don't you think ?